xpti/include/xpti/xpti_trace_framework.hpp

```diff
@@ -1613,6 +1622,10 @@ class tracepoint_t {
       }
     }
   }
+
+  tracepoint_t(const tracepoint_t &) = delete;
+  tracepoint_t &operator=(const tracepoint_t &) = delete;
```

Comment:
if a `tracepoint_t` object with `m_top` set to true is copied we would call `xptiSetUniversalId(xpti::invalid_uid)` twice on both of their destructions, so we likely do not want to be able to copy.